Refactor JavaScript callbacks example (#764)
Codecademy/docs: content/javascript/concepts/callbacks/callbacks.md

In JavaScript, functions can be passed as arguments because functions are objects.

> Suppose we want to create a function which makes a calculation and doubles the results of that calculation:

Reviewer: ```suggestion
Suppose a function is created which makes a calculation and doubles the results of that calculation:
```

We tend to avoid using words like `we` `you` `your` etc in the docs.

```js
function doubleResult(num1, num2, calc) {
  return calc(num1, num2) * 2;
}

function add(num1, num2) {
  return num1 + num2;
}

function multiply(num1, num2) {
  return num1 * num2;
}

console.log(doubleResult(4, 2, add)); // Output: 12
console.log(doubleResult(4, 2, multiply)); // Output: 16
```

> 

Reviewer: Can you add a code block of callback functions?

Author: Unfortunately I can't understand what you mean. Can you please clarify?

Reviewer: @rzvl Find more detail on Codebytes [here](https://github.com/Codecademy/docs/blob/6b01926359f5f1c3d3751a52f23e36b659c78f89/documentation/content-standards.md#codebytes-runnable-code-blocks).

Author: I just add a codebyte example. Hope it's what you asked for...

In JavaScript, functions can be passed as arguments because functions are objects.

Suppose a function is created which makes a calculation and doubles the results of that calculation:

```js
function doubleResult(num1, num2, calc) {
  return calc(num1, num2) * 2;
}

function add(num1, num2) {
  return num1 + num2;
}

function multiply(num1, num2) {
  return num1 * num2;
}

console.log(doubleResult(4, 2, add)); // Output: 12
console.log(doubleResult(4, 2, multiply)); // Output: 16
```

In the code above, `doubleResult` accepts three arguments `num1`, `num2`, and `calc`. `num1` and `num2` are the numbers which the user wants to be used in the calculation. `calc` is a function which the user passes as the third argument into the `doubleResult` function and contains the calculation code.

> `add` and `multiply` functions are just two basic functions which add OR multiply the two numbers passed into them as arguments.

Reviewer 2: ```suggestion
`add` and `multiply` functions are two basic functions which add or multiply the two numbers passed into them as arguments.
```

> In the log statments, we envoke `doubleResult()` function and pass two numbers as first and second arguments. We pass `add` OR `multiply`, as the third argument, to `doubleResult()` and 12 OR 16 is logged to the console depending on the third argument.

Reviewer 2: ```suggestion
In the log statements, the `doubleResult()` function is invoked with numbers as first and second arguments. The function `add` or `multiply` is passed as the third argument.
```
